account_financial_report/wizard/trial_balance_wizard.py

```diff
@@ -67,6 +67,15 @@ class TrialBalanceReportWizard(models.TransientModel):
         comodel_name="account.account",
         help="Ending account in a range",
     )
+    grouped_by = fields.Selection(
+        selection=[("", "None"), ("analytic_account", "Analytic Account")],
```

Comment:
This is problematic on next versions, so please put `"none"` or leave it non required (as it is) without default and don't add the "None" option.